networks/cardano/tokens/a32e3bc48813962740beff8eee45e916cd57ce8454ddf48efb7e58724d4f4749.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "",
+    "decimals": 1,
+    "mldId": "cardano-MOGI-a32e3bc48813962740beff8eee45e916cd57ce8454ddf48efb7e58724d4f4749",
+    "orgName": "Mogi Cet",
+    "orgUrl": "https://www.mogicet.xyz/",
+    "tokenAddress": "a32e3bc48813962740beff8eee45e916cd57ce8454ddf48efb7e58724d4f4749",
```

Comment:
This token only has 1 holder: https://cexplorer.io/asset/asset1dxs7h9pm6tmscg47nwfhgqc3nas8r2ua0fchsd